src/main/java/com/woowacamp/soolsool/core/cart/repository/CartItemRepository.java

```diff
@@ -3,8 +3,17 @@
 import com.woowacamp.soolsool.core.cart.domain.CartItem;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 
 public interface CartItemRepository extends JpaRepository<CartItem, Long> {
 
     List<CartItem> findAllByMemberIdOrderByCreatedAtDesc(final Long memberId);
+
+    @Query("SELECT distinct c FROM CartItem c join fetch c.liquor where c.memberId= :memberId order by c.createdAt desc")
+    List<CartItem> findAllByMemberIdOrderByCreatedAtDescWithFetchJoin(final Long memberId);
```

Comment:
```suggestion
    List<CartItem> findAllByMemberIdOrderByCreatedAtDescWithLiquor(final Long memberId);
```

그냥 fetch join보다 무엇과 함께 가져오는지 드러나면 좋을 것 같아요